apps/docs/docs/how-to-guides/theming/global-tokens/colors.mdx

```diff
@@ -0,0 +1,401 @@
+---
+hide_table_of_contents: true
+---
+
+import ColorPalette from '@site/src/components/ColorPalette/ColorPalette'
+import TokensTable from '@site/src/components/TokensTable/TokensTable'
+
+# Colors
+
+The store palette with all the tones needed.
+
+### Main
+
+<ColorPalette
+  colors={{
+    '--fs-color-main-0': '#f1f2f3',
+    '--fs-color-main-1': '#e3e6e8',
+    '--fs-color-main-2': '#00419e',
+    '--fs-color-main-3': '#002c71',
+    '--fs-color-main-4': '#002155',
+  }}
+/>
+
+### Accent
+
+<ColorPalette
+  colors={{
+    '--fs-color-accent-0': '#f1f2f3',
+    '--fs-color-accent-1': '#e3e6e8',
+    '--fs-color-accent-2': '#74808b',
+    '--fs-color-accent-3': '#5d666f',
+    '--fs-color-accent-4': '#171a1c',
+  }}
+/>
+
+### Neutral
+
+<ColorPalette
+  colors={{
+    '--fs-color-neutral-0': '#ffffff',
+    '--fs-color-neutral-1': '#f1f2f3',
+    '--fs-color-neutral-2': '#e3e6e8',
+    '--fs-color-neutral-3': '#c7ccd1',
+    '--fs-color-neutral-4': '#9099a2',
+    '--fs-color-neutral-5': '#74808b',
+    '--fs-color-neutral-6': '#5d666f',
+    '--fs-color-neutral-7': '#171a1c',
+  }}
+/>
+
+---
+
+## Semantical tokens
+
+<TokensTable
```

Comment:
1. Could we move the "copy button" to the right? (I think will be better if we place it right next to the token)

2. What do you think of breaking them into different sections according to the `type`?

<img width="984" alt="image" src="https://user-images.githubusercontent.com/3356699/201289595-d7376630-196c-4123-97d5-6f4a116f80d3.png">